content/join-us/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Join Us"
```

Comment:
Do we want to add some sort of SEO juice to this page? This is pretty bare

Reply:
Cool, I added seo title and keywords, will do the same for the other 2 PRs/MRs